src/main/scala/org/broadinstitute/dsde/agora/server/webservice/AddHandler.scala

```diff
@@ -45,10 +46,10 @@ class AddHandler(authorizationProvider: AuthorizationProvider) extends Actor {
   }
 
   private def add(requestContext: RequestContext, agoraEntity: AgoraEntity, username: String): Unit = {
-    val authorizedEntity = authorizationProvider.authorizationsForEntity(Some(agoraEntity), username)
-    if (!authorizedEntity.authorization.canCreate) {
-      context.parent ! RequestComplete(BadRequest, AgoraError("You don't have permission to create items in " + agoraEntity.namespace))
-    }
+//    val authorizedEntity = authorizationProvider.authorizationsForEntity(Some(agoraEntity), username)
+//    if (!authorizedEntity.authorization.canCreate) {
+//      context.parent ! RequestComplete(BadRequest, AgoraError("You don't have permission to create items in " + agoraEntity.namespace))
+//    }
```

Comment:
This supports the write, and I'm going to roll right into working on that. I just want to leave this here for myself as a TODO hook to come back to